src/Control/Email/Email.php

```diff
@@ -289,8 +289,19 @@ public function setSwiftMessage($swiftMessage)
      */
     private function getDefaultFrom(): string
     {
-        $defaultFrom = $this->config()->get('admin_email');
-        if (!$defaultFrom) {
+        // admin_email can have a string or an array config
+        // https://docs.silverstripe.org/en/4/developer_guides/email/#administrator-emails
+        $adminEmail = $this->config()->get('admin_email');
+        if (is_array($adminEmail) && count($adminEmail) > 0) {
+            $defaultFrom = array_keys($adminEmail)[0];
```

Comment:
```suggestion
            $defaultFrom = array_key_first($adminEmail);
```

Reply:
Updated

Reply:
Reverted, PHP 7.1 (which 4.9 supports) doesn't have the array_key_first, it's new with php 7.3.0

Reply:
Ah, ok, I thought we dropped support below 7.4 now but didn't realise 4.9 still has it. All good.

Reply:
Yeah 4.10 was when older PHP support was dropped.